docs/conf.py

```diff
@@ -143,3 +146,16 @@
     python=('https://docs.python.org/3', None),
     python2=('https://docs.python.org/2', None),
 )
+
+# Add support for the unreleased "next-version" change notes
+extensions += ['sphinxcontrib.towncrier']
+
+
+# -- Options for towncrier_draft extension --------------------------------------------
+
+PROJECT_ROOT_DIR = Path(__file__).parents[1].resolve()
+
+towncrier_draft_autoversion_mode = "draft"  # or: 'sphinx-release', 'sphinx-version'
+towncrier_draft_include_empty = True
+towncrier_draft_working_directory = PROJECT_ROOT_DIR
+# Not yet supported: towncrier_draft_config_path = 'pyproject.toml'  # relative to cwd
```

Comment:
I'm annoyed that towncrier_draft requires all of these settings. Why aren't the defaults good enough?

Reply:
Yeah, I just didn't want to assume the working directory because cwd may depend on what you set in `changedir` in `tox.ini` effectively making this setup extremely fragile when the setting there is changed because the connection is highly unobvious. I guess I could attempt implementing some sort of an autodetection on the extension side but for now, it only implements a bare minimum w/o making assumptions.

Essentially, if I were to look into the config and see `..`, I'd assume it's relative to the dir with `conf.py` but it's relative to the process' current working directory.